src/lib/output/plugins/JavascriptIndexPlugin.ts

```diff
@@ -66,12 +66,18 @@ export class JavascriptIndexPlugin extends RendererComponent {
             rows.push(row);
         }
 
-        const fileName = Path.join(event.outputDirectory, 'assets', 'js', 'search.js');
-        const data =
-            `var typedoc = typedoc || {};
+        const jsFileName = Path.join(event.outputDirectory, 'assets', 'js', 'search.js');
+        const jsData =
+            `console.warn('[typedoc] search.js file is deprecated. Please, use search.json file in your custom theme instead.');
```

Comment:
I'd rather just remove this with the switch to `.json`. The search file is usually the biggest part of the deployed docs, so this change might otherwise nearly double the size of our output.

Reply:
I have removed js file generation.